### DISCUSSION TITLE
URLを初期状態に戻す方法を教えてください

### DISCUSSION THREAD

**Discussion by sakurakun999** [community/community#113509] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

githubpagesを利用しているのですが、URLをカスタムドメインから初期の〜github.ioに戻す方法を教えてください。
問題は、昔アカウント自体にドメインをつけていたのですが、手違いでドメインを手放してしまいドメインは使用できない状態なのですが、どこかの設定には残っているようで〜.net.（リポジトリ名）になってしまいます。アカウントのカスタムドメインの設定やリポジトリ自体の設定も見直しましたがなお戻す方法は見つからず。
戻す方法を知っている方がいらっしゃいましたら教えてください。

**Comment by sakurakun999** [comment, marked as answer]
github.ioのリポジトリの設定を変更することで解決できました。

**Comment by queenofcorgis** [comment]
Hi @sakurakun999 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
